packages/dev/core/src/Meshes/Builders/greasedLineBuilder.ts

```diff
@@ -113,8 +114,20 @@ export interface GreasedLineMeshBuilderOptions extends GreasedLineMeshOptions {
 export function CreateGreasedLineMaterial(name: string, options: GreasedLineMaterialOptions, scene: Nullable<Scene>) {
     scene = <Scene>(scene ?? EngineStore.LastCreatedScene);
 
-    const material = options.materialType === GreasedLineMeshMaterialType.MATERIAL_TYPE_PBR ? new PBRMaterial(name, scene) : new StandardMaterial(name, scene);
-    new GreasedLinePluginMaterial(material, scene, options);
+    let material;
+    switch (options.materialType) {
+        case GreasedLineMeshMaterialType.MATERIAL_TYPE_PBR:
+            material = new PBRMaterial(name, scene);
+            new GreasedLinePluginMaterial(material, scene, options);
+            break;
+        case GreasedLineMeshMaterialType.MATERIAL_TYPE_SIMPLE:
+            material = new GreasedLineSimpleMaterial(name, scene, options);
+            break;
+        default:
+            material = new StandardMaterial(name, scene);
+            new GreasedLinePluginMaterial(material, scene, options);

```

Comment:
Same here

Reply:
It gets disposed automatically when the material which the plugin material is attached to is disposed. Every plugin material does.